docs/reST/ref/math.rst

```diff
@@ -445,6 +445,42 @@ Multiple coordinates can be set using slices or swizzling
 
       .. ## Vector2.update ##
 
+   
+   .. attribute:: epsilon
+
+      .. _Vector2:
+
+      | :sl:`Determines the tolerance of vector calculations.`
+      
+      Both Vector classes have a value named ``epsilon`` that defaults to ``1e-6``.
+      This value acts as a numerical margin in various methods to account for floating point
+      arithmetic errors. Specifically, ``epsilon`` is used in the following places:
+
+      * comparing Vectors (``==`` and ``!=``)
+      * the ``is_normalized`` method (if the square of the length is within ``epsilon`` of 1, it's normalized)
+      * slerping (a Vector with a length of ``<epsilon`` is considered a zero vector, and can't slerp with that)
+      * reflection (can't reflect over the zero vector)
+      * projection (can't project onto the zero vector)
+      * rotation (only used when rotating by a multiple of 90 degrees)
```

Comment:
When I generate this locally the bulleted list doesn't appear indented at all. Maybe giving it some more spaces on the left would fix this?